src/components/login_header.tsx

```diff
@@ -1,22 +1,25 @@
 import * as React from "react"
 import styled from "styled-components"
-import Icon from "./icons"
+import Icon from "./icon"
+import Title from "./title"
 
 const Header = styled.header`
-  margin: 20px;
+  margin: 20px auto;
   font-size: 26px;
   line-height: 1.3;
   text-align: center;
+  width: 400px;
```

Comment:
Good point. Unless it is guaranteed that a component is _always_ going to be a certain size, it should refrain from specifying any of its own metrics and leave that up to the user.